composer.json

```
@@ -39,136 +39,27 @@
         "mnsami/composer-custom-directory-installer": "^2.0"
     },
     "require-dev": {
-        "acquia/coding-standards": "^1.0",
-        "axelerant/drupal-quality-checker": "^1.1",
-        "drupal/core-composer-scaffold": "^9.0.0 || ^10",
-        "drupal/core-dev": "^9 || ^10",
-        "ergebnis/composer-normalize": "~2.15.0",
+        "acquia/coding-standards": "^1.0 || ^2.0",
+        "axelerant/drupal-quality-checker": "^1.3",
+        "drupal/core-composer-scaffold": "^10.2",
+        "drupal/core-dev": "^9 || ^10.2",
+        "ergebnis/composer-normalize": "~2.40.0",
         "mglaman/drupal-check": "^1.4",
         "oomphinc/composer-installers-extender": "^1.1 || ^2",
-        "phpspec/prophecy-phpunit": "^2.0",
-        "phpunit/phpunit": "~9.4.0 || ~9.5.0",
+        "phpspec/prophecy-phpunit": "^2",
+        "phpunit/phpunit": "~9.4.0 || ~9.5.0 || ~9.6.0 || ^10",
```

can we use it like this 
```suggestion
        "phpunit/phpunit": "^9.4 || ^10",
```